gensim/models/fastsent.py

```diff
@@ -0,0 +1,957 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2013 Radim Rehurek <me@radimrehurek.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+
+"""
+FastSent model for sentence representation.
+
+The FastSent algorithm was originally inspired by the C package https://code.google.com/p/fastsent/.
+
+For a blog tutorial on gensim fastsent, which is closely related, visit http://radimrehurek.com/2014/02/fastsent-tutorial/
```

Comment:
Hmm, this is from the original PR: https://github.com/piskvorky/gensim/pull/612/files

Perhaps they meant to link to this post? http://rare-technologies.com/word2vec-tutorial/